cli/sawtooth_cli/main.py

```diff
@@ -126,7 +126,7 @@ def main(prog_name=os.path.basename(sys.argv[0]), args=sys.argv[1:],
     elif args.command == 'cluster':
         do_cluster(args)
     else:
-        raise AssertionError("invalid command: {}".format(args.command))
```

Comment:
I've made more changes than I expected, to clean up the user experience of the sawtooth command. Lots of commands/subcommands produced ungraceful errors. These have been fixed.